packages/expandable/src/ux-expandable.ts

```diff
@@ -0,0 +1,76 @@
+import { customElement, useView, PLATFORM, bindable, TaskQueue, inject } from 'aurelia-framework';
+import { UxComponent, StyleEngine, normalizeBooleanAttribute } from '@aurelia-ux/core';
+import { UxExpandableTheme } from './ux-expandable-theme';
+
+@inject(Element, StyleEngine, TaskQueue)
+@customElement('ux-expandable')
+@useView(PLATFORM.moduleName('./ux-expandable.html'))
+export class UxExpandable implements UxComponent {
+  constructor(public element: HTMLElement, private styleEngine: StyleEngine, private taskQueue: TaskQueue) { }
+
+  static OPEN_CHANGED = 'open-changed';

```

Comment:
the naming of this should indicate what it is better. We can either:
- export an enum for all event: 
```ts
export const enum UxExpandableEvent {
  OpenChange = 'open-changed';
}
```

or change this name to `OPEN_CHANGED_EVENT`. Though I think a separate event export could probably be better. Not 100% sure on this